runtime/common/src/precompile/mock.rs

```diff
@@ -262,16 +239,16 @@ impl module_transaction_payment::Config for Test {
 	type Currency = Balances;
 	type MultiCurrency = Currencies;
 	type OnTransactionPayment = ();
-	type TransactionByteFee = TransactionByteFee;
-	type OperationalFeeMultiplier = OperationalFeeMultiplier;
-	type TipPerWeightStep = TipPerWeightStep;
-	type MaxTipsOfPriority = MaxTipsOfPriority;
-	type AlternativeFeeSwapDeposit = ExistentialDeposit;
+	type TransactionByteFee = ConstU128<10>;
+	type OperationalFeeMultiplier = ConstU64<5>;
+	type TipPerWeightStep = ConstU128<1>;
+	type MaxTipsOfPriority = ConstU128<1000>;
+	type AlternativeFeeSwapDeposit = ConstU128<1>;
 	type WeightToFee = IdentityFee<Balance>;
 	type FeeMultiplierUpdate = ();
 	type DEX = DexModule;
 	type MaxSwapSlippageCompareToOracle = MaxSwapSlippageCompareToOracle;
-	type TradingPathLimit = TradingPathLimit;
+	type TradingPathLimit = ConstU32<4>;
```

Comment:
`TradingPathLimit` is used by different configs, should keep it defined with `parameter_types!`.